### PR TITLE
Add tags for notification type and severity

### PR DESCRIPTION
**Pull request by steveburton4**
Add tags for notification type and severity

### CODE REVIEW

**Review by ChrisScottThomas — DISMISSED** [review]

File: main.tf [hunk 1]
```diff
@@ -20,4 +20,5 @@ resource "aws_cloudwatch_metric_alarm" "metric_alarm" {
   threshold           = var.threshold
   statistic           = var.statistic
   alarm_actions       = var.alarm_action_arns
+  tags                = {"notification_type" = var.notification_type, "severity" = var.severity}
```
Comment (dan-hill2802) [inline main.tf]: keys don't nee quotes

File: variables.tf [hunk 2]
```diff
@@ -70,4 +70,16 @@ variable "statistic" {
   type        = string
   description = "The statistic to apply to the alarm's associated metric. Either of the following is supported: SampleCount, Average, Sum, Minimum, Maximum"
   default     = "Sum"
-}
\ No newline at end of file
+}
+
+variable "notification_type" {
+  type        = string
+  description = "The notification type of this alarm. Either of the following is supported: Info, Warning, Error"
```
Comment (dan-hill2802) [inline variables.tf]:
```suggestion
  description = "(optional) The notification type of this alarm. Either of the following is supported: Info, Warning, Error"
```

File: variables.tf [hunk 3]
```diff
@@ -70,4 +70,16 @@ variable "statistic" {
...
+  default     = "Warning"
+}
+
+variable "severity" {
+  type        = string
+  description = "The severity of this alarm. Either of the following is supported: Low, Medium, High, Critical"
```
Comment (dan-hill2802) [inline variables.tf]:
```suggestion
  description = "(optional) The severity of this alarm. Either of the following is supported: Low, Medium, High, Critical"
```

File: main.tf (hunk 1 above)
Comment (steveburton4) [reply main.tf]: Updated

**Review by ChrisScottThomas — APPROVED** [review]

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**